examples/governance/apps/README.md

```diff
@@ -0,0 +1,88 @@
+# Apps
+
+In typical applications, an off-chain app is needed to do two main actions:
+
+* Produce a proof (see [proving options]).
```

Comment:
Ah now I understand, thanks for that. I know Nuke had some discussions with Frank about [lychee](https://github.com/lycheeverse/lychee) and that's now used in the main risc0 monorepo CI iirc, perhaps for consistency, it would make sense to use it in this repo as well